sources/.docker/postgres/create_db.sh

```diff
@@ -18,4 +18,4 @@ if [ -n "$POSTGRES_MULTIPLE_DATABASES" ]; then
     create_user_and_database $db
   done
   echo "Multiple databases created"
-fi
+fi
```

Comment:
Please avoid adding this to the commit if it is not necessary.

Reply:
Unable to